apps/web/app/(app)/(onboarding)/environments/[environmentId]/xm-templates/lib/xm-templates.ts

```diff
@@ -19,7 +16,7 @@ export const getXMSurveyDefault = (t: TFnType): TXMTemplate => {
       },
     };
   } catch (error) {
-    logError(error, "getXMSurveyDefault");
+    logger.warn(error, "getXMSurveyDefault");
```

Comment:
why did you decide to log this as warning? since it's an error that gets thrown I think `error` is a more appropriate level.

Reply:
Got it. I initially thought that the error should require immediate fixation, and these services can fail for many reasons that may not require any code fix. But the above summary of levels makes sense.